SignalServiceKit/src/Messages/OWSBatchMessageProcessor.m

```diff
@@ -338,7 +341,7 @@ - (void)drainQueueWorkStep
     // This delay won't affect the first message to arrive when this queue is idle,
     // so by definition we're receiving more than one message and can benefit from
     // batching.
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1f * NSEC_PER_SEC)), self.serialQueue, ^{
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1.f * NSEC_PER_SEC)), self.serialQueue, ^{
```

Comment:
Doesn't this max us out at 16 messages/second vs the previous 100 messages/second? That seems unnecessarily slow e.g. for startup with a big backlog.

Reply:
How about 32/0.5 seconds = 64 messages/sec.?  That's 4x faster.  Will churn through 1,000 messages in 15 seconds.

Reply:
That sounds more reasonable - we can continue to hone this as we go.